src/collective/contact/core/tests/test_utils.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf8 -*-
+import unittest2 as unittest
```

Comment:
we could start to follow plone's styleguide for new modules or the ones we touch in pull requests (https://docs.plone.org/develop/styleguide/python.html) and sort imports here.